content.js

```diff
@@ -20,7 +24,21 @@ function toggleFiles() {
 
 	for (const el of rows) {
 		if (el.querySelector('.content a[title^="."')) {
```

Comment:
No, read the code again. The if statement here checks if the title starts with `.` and only runs the regex if so.